### PR TITLE
Resolved some coding standard issue in the modules and themes file

### PR DESCRIPTION
**Pull request by VighneshSadagopal**
Resolved some coding standard issue for modules and themes folder.

### CODE REVIEW

File: web/modules/custom/log_mod/src/Controller/TestController.php [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-        '#markup' => 'Hello World',
-    );
+    $element = [
+      '#markup' => 'Hello World',
+    ];
     return $element;
   }
+
 }
-?>
```
Comment (sharayu0) [inline web/modules/custom/log_mod/src/Controller/TestController.php:12]: Add new line at the end of file after ?> tag and also check for remaining files

File: web/modules/custom/demo_con/demo_con.info.yml [hunk 2]
```diff
@@ -4,4 +4,4 @@ description: 'Default content export'
 core_version_requirement: ^8 || ^9
 package: "SDNN"
 dependencies:
-  - default_content:default_content
```
Comment (sharayu0) [inline web/modules/custom/demo_con/demo_con.info.yml:7]: Add new line at the end of file and also check for remaining files

File: web/modules/custom/log_mod/log_mod.module [hunk 3]
```diff
@@ -1,103 +1,98 @@
...
+        'class' => 'pass-toggle show',
+      ],
+      '#title' => t('Show'),
+    ];
+
+    $form['my-empty-container'] = [
+      '#type' => 'container',
+      '#attributes' => [
+        'class' => 'my-class',
+      ],
```
Comment (JatinGupta40) [inline web/modules/custom/log_mod/log_mod.module:46]: The brace should move 2 steps to the left (straight below the $ of line 42).

**Review by JatinGupta40 — CHANGES_REQUESTED** [review]

**Review by sharayu0 — APPROVED** [review]

File: web/modules/custom/log_mod/log_mod.module [hunk 4]
```diff
@@ -1,103 +1,98 @@
...
 
+/**
+ * @file
+ * To alter the Login form and add few more fields.
+ */
+
+use Drupal\Core\Url;
+
+/**
+ * Returns the variable from the custom block.
```
Comment (guptahemant) [inline web/modules/custom/log_mod/log_mod.module:11]: Hook implementations needs a different functional doc, Please add them as per official documentation.

File: web/modules/custom/log_mod/log_mod.module [hunk 5]
```diff
@@ -1,103 +1,98 @@
...
+      'variables' => [
+        'data' => NULL,
+      ],
+    ],
+  ];
 }
 
-
+/**
+ * Alter the login form.
```
Comment (guptahemant) [inline web/modules/custom/log_mod/log_mod.module:24]: Same here.

File: web/modules/custom/profile_block/profile_block.module [hunk 6]
```diff
@@ -1,34 +1,42 @@
 <?php
- 
+
+/**
+ * @file
+ * To store the variables that are been displayed on the profile page.
+ */
+
+/**
+ * To store the variable in the user profile block.
```
Comment (guptahemant) [inline web/modules/custom/profile_block/profile_block.module:9]: Same here.

**Pull request merged**